2_adventures/001/kotp/04_automated_testing_example/spec/fizz_buzz_spec.rb

```diff
@@ -0,0 +1,53 @@
+require_relative 'spec_helper'
+require 'fizz_buzz'
+
+describe FizzBuzz do
+  describe 'Single Numbers' do
+    it 'must return 1 for 1' do
```

Comment:
And I think I would actually prefer the `assert actual, expectation` form of this... rather than how I have it now.  It would of course report the same story in the report, but the testing reads cleaner.

So to illustrate:
...

New commit coming in... :)